relay-server/src/services/processor.rs

```diff
@@ -1212,6 +1212,25 @@ impl EnvelopeProcessorService {
         &self,
         state: &mut ProcessEnvelopeState<G>,
     ) -> Result<(), ProcessingError> {
+        let full_normalization = match self.inner.config.normalization() {
+            NormalizeEvents::Disabled => {
+                // We assume envelopes coming from an internal relay have
+                // already been normalized.  During incidents, envelopes can be
+                // routed differently, skipping some relays and reaching another
+                // internal one. Events should be fully normalized in all cases,
+                // so we force normalization even if it's disabled.
+                if self.inner.config.processing_enabled()
+                    && !state.envelope().meta().is_from_internal_relay()
+                {
+                    true
+                } else {
+                    return Ok(());
+                }
```

Comment:
Sorry for going in circles here, but I'm in favor of removing the extra check and simply skipping normalization unconditionally, as discussed with @jan-auer. We just need to make sure that we track down any playbooks that suggest sending data straight to processing Relays before we enable this config in production.

Reply:
After discussing this, we're going forward with this approach -- mostly due to increased reliability, minimal complexity increase, and it can be updated in the future with minimal breaking changes*.

*Breaking changes would only impact Self-Hosted instances with at least two layers of relays and where the last layer of relay also gets traffic (not just the outer ones).